src/resource.cpp

```diff
@@ -165,6 +166,11 @@ void Resource::readEntries() {
 */
 void Resource::loadMarkedAsNeeded() {
 
+	static const char *demofile = "demo3.joy";
```

Comment:
Since this file doesn't appear to be needed anywhere else, would it make sense to directly check if the `bank0c` file exists instead?

Reply:
in theory it is not needed for either DEMO or FULL versions of DOS ports (at least original rewrite info indicates so), but since later fails to run without it (in `rawgl` and Fabien's `raw` it crashes after awhile ingame) we could assume FULL relies on it in src.

It also worth to mention DEMO lacks ScreenProtection resources.